### ISSUE TITLE
UnusedMethod: False positive when case sensitivity doesn't match signature

### ISSUE THREAD

**Issue by Majkl578** [slevomat/coding-standard#206]
```
class Test
{
    public function doSomething() : void
    {
        $this->testfoo();
    }

    private function testFoo() : void
    {
    }
}
```
In this case, testFoo is deemed unused, although it is, but with incorrect case (that should be a task for separate sniff or PHPStan).
This happens with v3.3.0, haven't found any ticket that'd fix it in newer versions.
